compiler/src/iree/compiler/Codegen/Common/test/convolution_match_spec.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: iree-opt %s
+
+transform.sequence failures(propagate) {
+^bb0(%arg0: !pdl.operation):
+  transform.iree.register_match_callbacks
+
+  %fill, %convolution, %trailing =
+    transform.iree.match_callback failures(propagate) "convolution"(%arg0)
+    : (!pdl.operation) -> (!pdl.operation, !pdl.operation, !pdl.operation)
+
+  transform.iree.emit_remark "fill" at %fill : !pdl.operation
+  transform.iree.emit_remark "convolution" at %convolution : !pdl.operation
+  transform.iree.emit_remark "trailing" at %trailing : !pdl.operation
+}
```

Comment:
can this be merged with the file below?

Reply:
I'm mirroring what is in-tree for reductions right now: https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Codegen/Common/test/reductions.mlir

The idea is that we'll want to split it out later anyway once codegen strategies are added.